lib/node-repo.js

```diff
@@ -26,16 +26,23 @@ function resolveLabelsThenUpdatePr (options) {
     const filepathsChanged = res.map((fileMeta) => fileMeta.filename)
     const resolvedLabels = resolveLabels(filepathsChanged, options.baseBranch)
 
-    fetchExistingLabels(options, (err, existingLabels) => {
-      if (err) {
-        return options.logger.error(err, 'Error retrieving existing repo labels')
-      }
+    fetchExistingThenUpdatePr(options, resolvedLabels)
+  })
+}
+
+function fetchExistingThenUpdatePr (options, labels) {
+  fetchExistingLabels(options, (err, existingLabels) => {
+    if (err) {
+      options.logger.error(err, 'Error retrieving existing repo labels')
+
+      updatePrWithLabels(options, labels)
```

Comment:
Is this intentional? Previously on [L31](https://github.com/nodejs/github-bot/pull/103/files#diff-3f74e33a82dabff73070dddacecc4dadL31) we didn't label when an error occurred